daemon/config/config.go

```diff
@@ -32,8 +32,8 @@ const (
 	// maximum number of attempts that
 	// may take place at a time for each pull when the connection is lost.
 	DefaultDownloadAttempts = 5
-	// DefaultShmSize is the default value for container's shm size
-	DefaultShmSize = int64(67108864)
+	// DefaultShmSize is the default value for container's shm size (64 MiB)
+	DefaultShmSize int64 = 67108864
```

Comment:
Is it worth maybe changing this so that the "64 MiB" gets even more obvious since it'll be calculated at compile-time anyhow?

```suggestion
	DefaultShmSize int64 = 64 * 1024 * 1024
```

Reply:
Yes! Good suggestion. It's a `const` indeed, so doing so shouldn't make a difference (other than being more clear).

I actually went looking "doesn't go stdlib not have some pre-defined constants for this? But I found some discussion on golang-nuts, and it was (reasonably) rejected for stdlib itself (but could be in one of the `/x/` projects).